compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/try-catch-mutate-outer-value.expect.md

```diff
@@ -28,9 +28,9 @@ import { c as _c } from "react/compiler-runtime";
 const { shallowCopy, throwErrorWithMessage } = require("shared-runtime");
 
 function Component(props) {
-  const $ = _c(3);
+  const $ = _c(5);
   let x;
-  if ($[0] !== props.a) {
+  if ($[0] !== props) {
```

Comment:
Note that this is also a bugfix. I can change the fixture slightly and produce a difference in the evaluator

```js
function useHook(maybeNull) {
  const x = [];
  try {
    x.push(2);
  } catch {
    x.push(shallowCopy(maybeNull.value));
  }
  return x;
}
```